e2e-tests/uat.env

```diff
@@ -1,3 +1,4 @@
+APIM_HOST="https://api.uat.platform.pagopa.it"
 PM_HOST="https://api.uat.platform.pagopa.it"
```

Comment:
Could PM_HOST be removed?

Reply:
good catch! removed with https://github.com/pagopa/pagopa-wallet-tests/pull/44/commits/62d187e0db7971e5103d9a31c9f8cf6b777bcc0d